core-plugin/testSrc/com/google/gct/idea/appengine/validation/MethodParameterTypeInspectionTest.java

```diff
@@ -24,6 +24,14 @@
  */
 public class MethodParameterTypeInspectionTest extends EndpointTestBase {
 
+  @Override
```

Comment:
This needs more investigation. It's not part of some (not all) of their downloads. Not yet clear why.